recipes/infer-subc/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "infer-subc" %}
+{% set version = "0.0.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/infer-subc-{{ version }}.tar.gz
+  sha256: 7ff6222f124d7167c3334e25d0178adc8d0205b3f5f189fb7ebc4e34379959a8
+
+build:
+  entry_points:
+    - infer_subc = infer_subc.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - aicsimageio
+    - aicssegmentation
+    - jupyter
+    - napari
+    - numpy
+    - pandas
+    - python >=3.8
+    - scikit-image
+    - scipy
+
+test:
+  imports:
+    - infer_subc
+    - infer_subc.core.img
+    - infer_subc.organelles
+    - infer_subc.workflow
+  commands:
+    - pip list
+    # - pip check  # Fails due to aicssegmentation and itkwidgets
+    - infer_subc --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ndcn/infer-subc
+  summary: A plugin that enables organelle segmentation
+  license: BSD-3-Clause
```

Comment:
```suggestion
  license: Unlicense
```

That is not the file that comes with the tar file for this version https://files.pythonhosted.org/packages/68/2b/9c2911b9204e97e949d7c327a5c9f19e2565956acf9582d31b15f0640a2c/infer-subc-0.0.post1.tar.gz

`LICENSE`

```
This is free and unencumbered software released into the public domain.

Anyone is free to copy, modify, publish, use, compile, sell, or
distribute this software, either in source code form or as a compiled
binary, for any purpose, commercial or non-commercial, and by any
means.

In jurisdictions that recognize copyright laws, the author or authors
of this software dedicate any and all copyright interest in the
software to the public domain. We make this dedication for the benefit
of the public at large and to the detriment of our heirs and
successors. We intend this dedication to be an overt act of
relinquishment in perpetuity of all present and future rights to this
software under copyright law.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
OTHER DEALINGS IN THE SOFTWARE.

For more information, please refer to <https://unlicense.org>
```

Reply:
This seems to be fixed in the latest release at least.